transport/node.py

```diff
@@ -35,13 +37,29 @@ def stop(self):
         """
 
     @abstractmethod
-    def send_message(self, message: Message, recipient: Address):
+    def enqueue_message(self, message: Message, recipient: Address):
         """
-        Send a message to the recipient.
+        Enqueue a message to be sent the recipient.
```

Comment:
```suggestion
        Enqueue a message to be sent to the recipient.
```

Reply:
We can add to the docstring something like 

"This method must be called by the raiden_service in order to send a message¨, just like the send_message clarification with the RetryQueue usage

Reply:
> We can add to the docstring something like
> 
> "This method must be called by the raiden_service in order to send a message¨, just like the send_message clarification with the RetryQueue usage

the way i see it: there's a difference with `send_message`; while the latter should _only_ be called by the `_RetryQueue` class, this one can arbitrarily be called from any other entity.

in our case, both the `RaidenService` and the `RaidenEventHandler` classes make use of this function (through reference to transport nodes)